tasks/trace_agent.py

```diff
@@ -55,15 +53,11 @@ def build(
 
     # generate windows resources
     if sys.platform == 'win32':
-        if arch == "x86":
-            env["GOARCH"] = "386"
-
-        build_messagetable(ctx, arch=arch)
-        vars = versioninfo_vars(ctx, major_version=major_version, python_runtimes=python_runtimes, arch=arch)
+        build_messagetable(ctx)
```

Comment:
Does this whole section need to stay? Or are we removing the ability to use `win32` as a platform too?

Reply:
IIRC `win32` = Windows in Python (it does not just mean Windows 32 bits). Since we're not dropping Windows support, I just removed the `if` block !